### PR TITLE
Test fixture for terraform-google-jenkins added

### PR DESCRIPTION
**Pull request by ingwarr**

### CODE REVIEW

File: infra/terraform/test_fixtures/terraform-google-jenkins.tf [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "roles/compute.networkAdmin",
+    "roles/iam.serviceAccountUser",
+  ]
+}
+
+resource "google_project" "jenkins" {
+  provider = "google.phoogle"
+
+  name            = "ci-jenkins"
+  project_id      = "ci-jenkins"
```
Comment (morgante) [inline infra/terraform/test_fixtures/terraform-google-jenkins.tf]: This name isn't unique - choose something globally unique, like `phoogle-ci-jenkins`.

Comment (ingwarr) [reply infra/terraform/test_fixtures/terraform-google-jenkins.tf]: Ok, thank you, I'll change it

File: infra/terraform/test_fixtures/terraform-google-jenkins.tf [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  vars {
+    pipeline      = "terraform-google-jenkins"
+    webhook_token = "${random_id.jenkins_github_webhook_token.hex}"
+  }
+}
+
+resource "kubernetes_secret" "jenkins" {
+  metadata {
+    namespace = "concourse-cft"
+    name      = "phoogle-jenkins"
```
Comment (aaron-lane) [inline infra/terraform/test_fixtures/terraform-google-jenkins.tf]: This name must match the name used in the Concourse pipeline.

```suggestion
    name      = "jenkins"
```

File: infra/concourse/build/Dockerfile.kitchen-terraform [hunk 3]
```diff
@@ -35,6 +35,12 @@ SHELL ["/bin/bash", "-c"]
 
 ENV APP_BASE_DIR="/cft"
 
+RUN cd /tmp && \
+    wget https://releases.hashicorp.com/packer/1.3.3/packer_1.3.3_linux_amd64.zip && \
```
Comment (aaron-lane) [inline infra/concourse/build/Dockerfile.kitchen-terraform]: Let's use the latest release of Packer, 1.4.1.

Comment (ingwarr) [reply infra/concourse/build/Dockerfile.kitchen-terraform]: Ok, I'll change it, thank you!

File: infra/terraform/test_fixtures/terraform-google-jenkins.tf (hunk 2 above)
Comment (ingwarr) [reply infra/terraform/test_fixtures/terraform-google-jenkins.tf]: Done

File: infra/concourse/build/Dockerfile.kitchen-terraform (hunk 3 above)
Comment (ingwarr) [reply infra/concourse/build/Dockerfile.kitchen-terraform]: Done

File: infra/concourse/build/Dockerfile.kitchen-terraform [hunk 4]
```diff
@@ -36,9 +36,9 @@ SHELL ["/bin/bash", "-c"]
 ENV APP_BASE_DIR="/cft"
 
 RUN cd /tmp && \
-    wget https://releases.hashicorp.com/packer/1.3.3/packer_1.3.3_linux_amd64.zip && \
-    unzip packer_1.3.3_linux_amd64.zip && \
-    rm -rf packer_1.3.3_linux_amd64.zip && \
+    wget wget https://releases.hashicorp.com/packer/1.4.1/packer_1.4.1_linux_amd64.zip && \
```
Comment (aaron-lane) [inline infra/concourse/build/Dockerfile.kitchen-terraform]: This duplicate `wget` must be removed:

```suggestion
    wget https://releases.hashicorp.com/packer/1.4.1/packer_1.4.1_linux_amd64.zip && \
```

**Pull request merged**